### ISSUE TITLE
ibdsim exits immediately upon start on 6d4a25fde

### ISSUE THREAD

**Issue by kcalvinalvin** [mit-dci/utreexo#140]
log:

```
Creating new pollarddata
Block 1 add 0 del 0 pol nl 0 roots 0 he 0 re 0 ow 0
plus 0.00 total 0.00
Pollard leaves: 0
Done Writing
```

This effect is not seen on 6edfeb553dc23

**Comment by dergoegge** [comment]
hm, is there any suspicious output from genproofs?
my best guess is that genproofs closed the connection for some reason, since that is what 6d4a25fde handles.

**Comment by kcalvinalvin** [comment]
```
pushBlocks ub.Serialize write tcp 127.0.0.1:8338->127.0.0.1:45418: write: connection reset by peer
````

This is the log by genproofs

**Comment by adiabat** [comment]
Fixed in 4e8129e0c2dd0df6ca1905d603fb96c1cb89dfd2 ( v, ok := <- channel -- ok means the channel is open, not closed)
